lib/prawn/text/formatted/parser.rb

```diff
@@ -238,6 +236,14 @@ def self.array_from_tokens(tokens)
           end
           array
         end
+
+        def self.escape(text)
+          text.gsub('&', '&amp;').gsub('>', '&gt;').gsub('<', '&lt;')
+        end
+
+        def self.unescape(text)
+          text.gsub('&lt;', '<').gsub('&gt;', '>').gsub('&amp;', '&')
```

Comment:
Additionally, we may move the mapping elsewhere to make sure that `escape` and `unescape` perform mirror transformations.